src/utils/utils/utils.ts

```diff
@@ -16,3 +16,17 @@ export const generateName = (prefix: string): string => {
 
 export const getValidNamespace = (activeNamespace: string) =>
   activeNamespace === ALL_NAMESPACES_KEY ? DEFAULT_NAMESPACE : activeNamespace;
+
+export const get = (obj: unknown, path: string | string[], defaultValue = undefined) => {
+  const travel = (regexp: RegExp) =>
+    String.prototype.split
```

Comment:
this is risky. why?

Reply:
what this fundtions does? its not easy to understand

Reply:
It's the same one we have in the kubevirt-plugin. It replicates the functionality in the lodash version.

Reply:
I read it wrong, though u are overriding the prototype. this is not the case all good